ros/resources/v0/hosts.py

```diff
@@ -31,7 +31,7 @@ class HostsApi(Resource):
         'account': fields.String,
         'recommendation_count': fields.Integer,
         'state': fields.String,
-        'performance_score': fields.Nested(performance_score_fields),
+        'display_performance_score': fields.Nested(display_performance_score_fields),
```

Comment:
Thanks for the suggestions @kgaikwad, made all the changes accordingly.
But, there is still one thing that I'm guessing, since the field name is not `performance_profile` now, it doesn't show the records on UI.
Anything specific about this or I'm missing something here? Could you test it locally on UI?